src/main/webapp/i18n/de/aboutUs.json

```
@@ -68,7 +68,9 @@
             "getInTouch": "Kontakt",
             "contributors": "Mitwirkende",
             "projectManager": "Projekt Manager",
-            "website": "Webseite"
+            "website": "Webseite",
+            "administrator": "System Administrator",
+            "exerciseManagement": "Übungsleitung"
```

```suggestion
            "roles": {
                "projectManager": "Projektmanagement",
                "administrator": "Systemadministration",
                "exerciseManagement": "Übungsleitung"
            }
```
A separate subgroup for the known options for contributor roles would make clear where/how they are used.

---

I definitely agree. I'll add this change.